source/Willow-Core/SingleSelectionWebView.class.st

```diff
@@ -0,0 +1,114 @@
+Class {
+	#name : #SingleSelectionWebView,
+	#superclass : #SingleSelectionWebViewBehavior,
+	#instVars : [
+		'commandToComponent',
+		'interactionInterpreter',
+		'identifierAssigner',
+		'optionCommand'
+	],
+	#category : #'Willow-Core-WebViews'
+}
+
+{ #category : #'instance creation' }
+SingleSelectionWebView class >> dropDownListApplying: aComponentCommand [
```

Comment:
Avoid unnecessary instance creation methods. All information should be passed when creating. If there are many cases with no command, create a separate message in the supplier, but not here.